samples/security/spire/README.md

```diff
@@ -0,0 +1,72 @@
+# Integrating SPIRE as a CA through Envoy's SDS API
```

Comment:
It would be good to add the steps in a test in the istio.io PR if possible. I think it would need to be in the set of tests where you install istio (as your require the spire Quickstart first).

Reply:
I believe @alexandrealvino is taking care of this.